src/plugins/data_source_management/public/components/direct_query_data_sources_components/direct_query_data_connection/manage_direct_query_data_connections_table.tsx

```diff
@@ -64,10 +69,51 @@ export const ManageDirectQueryDataConnectionsTable: React.FC<ManageDirectQueryDa
   savedObjects,
   uiSettings,
   featureFlagStatus,
+  application,
 }) => {
+  const [observabilityDashboardsExists, setObservabilityDashboardsExists] = useState(false);
+  const [showIntegrationsFlyout, setShowIntegrationsFlyout] = useState(false);
+  const [integrationsFlyout, setIntegrationsFlyout] = useState<React.JSX.Element | null>(null);
+
+  const checkIfObservabilityDashboardsPluginIsInstalled = () => {
+    fetch('/api/status', {
+      headers: {
+        'Content-Type': 'application/json',
+        'osd-xsrf': 'true',
+        'accept-language': 'en-US,en;q=0.9,zh-CN;q=0.8,zh;q=0.7,zh-TW;q=0.6',
+        pragma: 'no-cache',
+        'sec-fetch-dest': 'empty',
+        'sec-fetch-mode': 'cors',
+        'sec-fetch-site': 'same-origin',
+      },
+      method: 'GET',
+      referrerPolicy: 'strict-origin-when-cross-origin',
+      mode: 'cors',
+      credentials: 'include',
+    })
```

Comment:
Why do we use fetch here? There should be http.get from core service.

Reply:
Cuz we need to customize the headers. This is a internal call btw. It just check plugin installation status.

Reply:
But http.get should work as the same as well? I do not see there are any special fields in headers.